modules/apps/web-experience/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/page.jsp

```diff
@@ -27,13 +27,14 @@ String id = GetterUtil.getString((String)request.getAttribute("liferay-asset:ass
 PortletURL portletURL = (PortletURL)request.getAttribute("liferay-asset:asset-tags-selector:portletURL");
 String removeCallback = GetterUtil.getString((String)request.getAttribute("liferay-asset:asset-tags-selector:removeCallback"));
 String tagNames = GetterUtil.getString((String)request.getAttribute("liferay-asset:asset-tags-selector:tagNames"));
+	boolean changeStructure = GetterUtil.getBoolean(request.getAttribute("edit_article.jsp-changeStructure"));
 %>
 
 <div class="lfr-tags-selector-content" id="<portlet:namespace /><%= id %>assetTagsSelector">
 	<aui:input name="<%= hiddenInput %>" type="hidden" />
 
 	<c:if test="<%= allowAddEntry %>">
-		<input class="form-control lfr-tag-selector-input" id="<%= id %>assetTagNames" size="15" title="<liferay-ui:message key="add-tags" />" type="text" />
+		<textarea class="form-control lfr-tag-selector-textarea" id="<%= id %>assetTagNames" name="assetTagNames" title="<liferay-ui:message key="add-tags" />"></textarea>
```

Comment:
We need to use a portlet namespace/portlet id to differentiate the names of these fields since more than one can be added to a page, which would then cause a conflict when calling it below.